src/Compilers/CSharp/Test/Semantic/Semantics/SuppressAccessibilityChecksTests.cs

```diff
@@ -114,8 +114,8 @@ static void M()
 
             var expr = (ExpressionSyntax)tree.GetCompilationUnitRoot().DescendantNodes().OfType<SimpleLambdaExpressionSyntax>().Single().Body;
 
-            var symbolInfo = model.GetSpeculativeSymbolInfo(expr.FullSpan.Start, 
```

Comment:
I suspect these trailing spaces are introduced when people write 1-line invocation expressions and then split them into multiple lines. Maybe we can update the product to auto-remove these trailing spaces in this circumstance (enter leaving trailing space on the previous line). Filed #940 for this.